src/codeu/chat/client/simplegui/NewAccount.java

```diff
@@ -0,0 +1,74 @@
+package codeu.chat.client.simplegui;
+
+/**
+ * @author  Troy Jones
+ * @date    5/21/17
+ * @brief   Contains all GUI necessary to execute the frame to 
+ *          create a new account.
+ */
+
+import java.awt.Button;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+
+import javax.swing.BoxLayout;
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+
+@SuppressWarnings("serial")
+public class NewAccount extends JFrame {
+
+    // constructor
+    public NewAccount () {
+
+        // create mainPanel
+        JPanel mainPanel = new JPanel();
+        BoxLayout mainPanelLayout = 
+            new BoxLayout(mainPanel, BoxLayout.PAGE_AXIS);
```

Comment:
2 * indent

so, if your indent is 4 spaces, overflow should have 2*4 = 8 spaces 

as an example where indent is 2 spaces (represented by >)
>> if (foo
>>>>>> && foo) {
>>>> // code
>> }